src/wireguard_tools/__init__.py

```diff
@@ -7,8 +7,8 @@
 
 __version__ = "0.4.4.post.dev0"
 
-from .wireguard_config import WireguardConfig
+from .wireguard_config import WireguardConfig, WireguardPeer
 from .wireguard_device import WireguardDevice
 from .wireguard_key import WireguardKey
 
-__all__ = ["WireguardConfig", "WireguardDevice", "WireguardKey"]
+__all__ = ["WireguardConfig", "WireguardPeer", "WireguardDevice", "WireguardKey"]
```

Comment:
Only change I would suggest is to alphabetically sort the names in this list

Reply:
Done